src/main/java/com/endava/endabank/security/jwtauthentication/JwtAuthenticationEntryPoint.java

```diff
@@ -17,12 +17,13 @@
 @Component
 public class JwtAuthenticationEntryPoint implements AuthenticationEntryPoint {
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
     @Override
-    public void commence(HttpServletRequest request, HttpServletResponse response, AuthenticationException authException) throws IOException, ServletException {
+    public void commence(HttpServletRequest request, HttpServletResponse response, AuthenticationException authException) throws IOException {
         response.setStatus(HttpServletResponse.SC_UNAUTHORIZED);
         response.setHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE);
-        Map<String,String> dataResponse = new HashMap<>();
-        dataResponse.put("message",authException.getMessage());
-        OBJECT_MAPPER.writeValue(response.getOutputStream(),dataResponse);
+        Map<String, String> dataResponse = new HashMap<>();
```

Comment:
Hi,

You can replace the lines 25 and 26 for
 `Map<String, String> dataResponse = Map.of(Strings.MESSAGE_RESPONSE, authException.getMessage());`